projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java

```diff
@@ -706,6 +709,14 @@ public String initContainer(@Nullable String containerName, @Nullable String con
     if (!containerDir.toFile().mkdirs()) {
       throw new BatfishException("failed to create directory '" + containerDir + "'");
     }
+    Path testrigsDir = containerDir.resolve(BfConsts.RELPATH_TESTRIGS_DIR);
+    if (!testrigsDir.toFile().mkdir()) {
+      throw new BatfishException("failed to create directory '" + testrigsDir + "'");
+    }
+    Path analysesDir = containerDir.resolve(BfConsts.RELPATH_ANALYSES_DIR);
+    if (!analysesDir.toFile().mkdir()) {
+      throw new BatfishException("failed to create directory '" + analysesDir + "'");
+    }
     return containerName;
```

Comment:
I feel this makes more sense when we has Object representation of it. For example, a container object has a list of Analysis, I feel an empty list should map to an empty folder, and null map to non-exist folder. In this case, I think a null value doesn't make sense. Also, initialization of testrig now creates all needed paths, I think they should be consistency.